### PR TITLE
Fix data params format options

### PR DESCRIPTION
**Pull request by wm75**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [ ] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

**Comment by wm75** [comment]
The line break and the indentation in the token got expanded into the params format identifier string.
While Galaxy seems to cope with that when building the tool interface, it spoils the json you get under
/api/tools/spades?io_details=True and the association with an edam format identifier.

File: tools/spades/macros.xml [hunk 1]
```diff
@@ -1,6 +1,4 @@
 <macros>
     <token name="@TOOL_VERSION@">3.12.0</token>
-    <token name="@INTYPES@">
-        fasta,fasta.gz,fastq,fastq.gz,fastqsanger,fastqsanger.gz
-    </token>
+    <token name="@INTYPES@">fasta,fasta.gz,fastq,fastq.gz,fastqsanger,fastqsanger.gz</token>
```
Comment (mvdbeek) [inline tools/spades/macros.xml]: while we're at it let's also drop the redundant formats:
```suggestion
    <token name="@INTYPES@">fasta,fasta.gz,fastq,fastq.gz</token>
```

**Comment by bgruening** [comment]
@wm75 @mvdbeek shouldn't we fix this in Galaxy and strip the string? I have seen also `xxx, yyy` not working. Maybe this is related.

**Comment by mvdbeek** [comment]
Sure, but there's no harm merging this IMO.

**Comment by wm75** [comment]
> I have seen also `xxx, yyy` not working. Maybe this is related.
@bgruening not working in the sense of preventing edam format association?

**Comment by wm75** [comment]
@jmchilton This is the whitespace inside a parameter attribute issue I was mentioning briefly to you.
In your opinion, should planemo flag this as an error, or should Galaxy just sanitize the xml silently everywhere?

**Comment by bernt-matthias** [comment]
Are the formats really redundant?

**Comment by wm75** [comment]
Why not? Isn't fastq including fastqsanger? Then there is no need to accept fastqsanger explicitly if fastq is accepted.

**Comment by bernt-matthias** [comment]
Not sure. I always forget if this works. Can you try? Maybe with a test or try manually with `planemo server`.

**Comment by wm75** [comment]
spades actually already has tests for all the different input formats, e.g., https://github.com/galaxyproject/tools-iuc/blob/137c64bd71994d77c8b90c5d6504111e037d94b4/tools/spades/spades.xml#L300
:smile:

**Comment by mvdbeek** [comment]
I don't think we need to bump as it doesn't change the output and is more of a cosmetic fix (except when you want to parse formats from the /api/tools/<tools_id>?io_details=true)

**Comment by wm75** [comment]
Yes, that was my thought, too.

**Comment by bernt-matthias** [comment]
I guess we can merge then. Thanks a lot.

**Comment by wm75** [comment]
underlying issue is addressed in https://github.com/galaxyproject/galaxy/pull/9022

**Pull request merged**